tools/validators/ontology_validator/yamlformat/tests/external_file_lib_test.py

```diff
@@ -32,7 +32,7 @@
 class ExternalFileLibTest(absltest.TestCase):
 
   def test_RecursiveDirWalk_oneLevel(self):
-    path_parts = external_file_lib._RecursiveDirWalk(DIR_ONE_LEVEL)
+    path_parts = external_file_lib.RecursiveDirWalk(DIR_ONE_LEVEL)
```

Comment:
since it is a public you need to add a unit test

Reply:
This is the unit test that already existed for `_RecursiveDirWalk`, refactored to be public.